src/components/OrgFile/components/HeaderContent/index.js

```diff
@@ -78,8 +78,10 @@ class HeaderContent extends PureComponent {
   }
 
   calculateRawDescription(header) {
-    // this is for display only, se we keep the default indentation behaviour
-    return createRawDescriptionText(header, false, false);
+    // This generates the text that appears in the description text field.
+    const dontIndent = this.state.base.get('shouldNotIndentOnExport');
+    // const dontIndent = this.props.dontIndent;
```

Comment:
@munen would you help me with this:
- I added `dontIndent` to the props (see below `mapStateToProps`).
- I can use the new prop (commented line)

But why can't I use the state directly? In the same file, `this.state` is used in many places. But using it here, I get an error, state is undefined. Maybe I used to understand this already, but I forgot it ^^

Reply:
@schoettl I'll happily try to help(; You cannot access `this.state` in `calculateRawDescription` for two reasons:

1. The local `state` is defined dependent on the `calculateRawDescription`: https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderContent/index.js#L46-L47
2. Only functions that are bound have access to `this` of the component (see https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderContent/index.js#L25).

Having said that, I don't think that you want to access the local state of the component, anyway. It's not the right place containing the information whether or not `dontIndent` is set. The `props` are the right place for that and you've already set that up correctly.

So, if I'm not mistaken, you want to use the props and not the state(;

Reply:
Thanks! That exactly answers my question.

Reply:
Happy to be of help :pray: